.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!--- Provide a general summary of your changes in the Title above -->
```

Comment:
Yea i think it really makes it easier and i hope you like the template docs i have for it